src/query-builder/function-module.ts

```diff
@@ -696,6 +723,10 @@ export interface FunctionModule<DB, TB extends keyof DB> {
         : never
   >
 
+  jsonAgg<RE extends StringReference<DB, TB>>(
+    column: RE,
+  ): RawBuilder<ExtractTypeFromStringReference<DB, TB, RE>[] | null>
```

Comment:
Why is a `RawBuilder` returned here and not `AggregateFunctionBuilder`?

Reply:
e.g. this is valid SQL:

```sql
SELECT name,
  json_agg(laptop.model order by laptop.model DESC) as models
from employee 
left join laptop on employee.empId = laptop.assignedTo 
group by employee.empId;
```
https://onecompiler.com/postgresql/435v97hnu

Reply:
It's due to my inexperience with this library 😅 Thanks for pointing this out. Should I add a query like this to the tests as well?

Reply:
> It's due to my inexperience with this library 😅

When you're new to a codebase, it's perfectly fine to not get things right the first time.

I'm just asking questions to get a feel for your thought process here.

You either have a good reason for your choice or by asking, I could find something that could be improved, so the next time, it's clearer what the correct choice should be.